test/test_sensu.rb

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env ruby
+require File.dirname(__FILE__) + '/helper'
+
+class TestSensu < Test::Unit::TestCase
+  def test_sensu_notify
+    sensu = God::Contacts::Sensu.new
+    sensu.check_name = "TestSensuContact"
+
+    sensu.notify("Test", Time.now, "Test", "Test", "")
+    assert_equal "sent sensu #{sensu.check_name} notification with status code #{sensu.status_code}", "sent sensu #{sensu.check_name} notification with status code #{sensu.status_code}"
```

Comment:
Instead of this `assert_equal`, you can use a mock before calling `sensu.notify()` to ensure that the right thing is sent to `UDPSocket`:

``` ruby
    UDPSocket.any_instance.expects(:send)
    sensu.notify("Test", Time.now, "Test", "Test", "")
```